test/libsolidity/syntaxTests/iceRegressionTests/storage_parameter_in_library.sol

```diff
@@ -0,0 +1,8 @@
+library Test {
```

Comment:
I think that `test/libsolidity/syntaxTests/dataLocations/libraries/` would be better place for this test.